contents/docs/libraries/shopify.mdx

```diff
@@ -0,0 +1,155 @@
+  posthog.init('<ph_project_api_key>',{api_host:'<ph_instance_address>'})
+  const checkout = event.data.checkout;
+
+  posthog.identify(checkout.email, {
+      email: checkout.email,
+      first_name: checkout.billingAddress.firstName,
+      last_name:  checkout.billingAddress.lastName,
+    }
+  )
+  posthog.capture('merch store order submitted', {
+    value: checkout.totalPrice.amount,
+    created_at: event.timestamp,
+    order_number: checkout.order.id,
+    userId: event.clientId,
+    order_id: checkout.order.id,
+    products: checkout.lineItems.map((item) => {
+      return {
+        product_id: item.id,
+        title: item.variant.product.title,
+        quantity: item.quantity,
+        price: item.variant.price.amount,
+        currency: item.variant.price.currencyCode,
+        sku: item.variant.sku,
+        size: item.variant.product.size
+      };
+    }),
+  })
+});
+```
+
+With this, every time a Shopify `checkout_completed` event fires (when a user completes a purchase), PostHog captures the event.
+
+<ProductScreenshot 
+  imageLight={eventLight} 
+  imageDark={eventDark} 
+  alt="Shopify events in PostHog" 
+/>
+
+## Installing PostHog in a Shopify theme
+
+To have unrestricted access to the features of PostHog in your Shopify store, you can install PostHog in your Shopify theme. To do this:
+
+1. Get your PostHog JavaScript snippet from [your project settings](https://us.posthog.com/settings/project#snippet)
+1. Login to your Shopify dashboard
+1. Go to 'Online Store' -> 'Themes' (see image below)
+1. On your theme, click 'Actions' -> 'Edit code' (see image below)
+
+![Shopify Dashboard](../../images/tutorials/shopify/shopify-dashboard.png)
+
+1. You should now be seeing a code editor. Click on `theme.liquid` under 'Layout' on the left sidebar (see image below)
+1. Navigate until you see the closing `</head>` tag. Paste your snippet there, before that tag, like in the image below:
+
+![Shopify Dashboard](../../images/tutorials/shopify/snippet.png)
+
+1. Click the green save button on the top right and you're good to go - PostHog should now capturing events on your Shopify store!
+
+> To confirm PostHog is configured correctly, visit your store and then check if the events from your session appear in PostHog. This may take a few minutes.
+
+### Advanced tracking
+
+It can be useful to track additional events on your Shopify store, such as when a user adds an item to their cart or completes a purchase.
+
+Although [autocapture](/docs/product-analytics/autocapture) does capture these events, you may need to add additional metadata. This information can be used in further analysis in aggregate or when understanding behavior of an individual user.
+
+> The following requires some knowledge of HTML and Shopify's [Liquid templating engine](https://shopify.dev/docs/api/liquid).
```

Comment:
```suggestion
> This section requires some knowledge of HTML and Shopify's [Liquid templating engine](https://shopify.dev/docs/api/liquid).

It can be useful to track additional events on your Shopify store, such as when a user adds an item to their cart or completes a purchase.

Although [autocapture](/docs/product-analytics/autocapture) does capture these events, you may need to add additional metadata. This information can be used in further analysis in aggregate or when understanding behavior of an individual user.
```